Probesys/bileto
# imp: Separate permissions to see real vs. accounted times spent

## Related issue(s)



https://github.com/Probesys/bileto/issues/559

## How to test manually



1. add 2 times spent with a user: one unaccounted, and the other one accounted for a contract
2. check both accounted and real times spent permissions
3. verify that you see the details of times spent
4. check only accounted times spent permission
5. verify that you see only the accounted time
4. check only real times spent permission
5. verify that you see only the real time

## Checklist
